### ISSUE TITLE
Use of CNVpytor in haploid organisms

### ISSUE THREAD

**Issue by pabloangulo7** [abyzovlab/CNVpytor#78]
Hello,

I would like to use this tool to examine the copy number variation in whole genome sequence, germline, haploid organism data. It would be possible? If so, which parameters should I modify?

Thanks

Pablo

**Comment by suvakov** [comment]
Yes, it is possible to use CNVpytor for haploid organisms. You have to configure reference genome (see [instructions](https://github.com/abyzovlab/CNVpytor/blob/master/examples/AddReferenceGenome.md)). For chromosome type specify "S" for haploid.
